pandas/tests/tslibs/test_parse_iso8601.py

```diff
@@ -47,18 +47,13 @@ def test_parsers_iso8601(date_str, exp):
     "20010101 12345Z",
 ])
 def test_parsers_iso8601_invalid(date_str):
-    msg = "Error parsing datetime string \"{s}\"".format(s=date_str)
-
-    with pytest.raises(ValueError, match=msg):
+    with pytest.raises(ValueError):
         tslib._test_parse_iso8601(date_str)
 
 
 def test_parsers_iso8601_invalid_offset_invalid():
     date_str = "2001-01-01 12-34-56"
-    msg = ("Timezone hours offset out of range "
```

Comment:
pls don't remove exception message checking, this is on purpose

Reply:
Rollback this